active-directory-b2c-wpf/TokenCacheHelper.cs

```diff
@@ -1,33 +1,5 @@
-﻿//------------------------------------------------------------------------------
-//
-// Copyright (c) Microsoft Corporation.
-// All rights reserved.
```

Comment:
nit: why are copyright notices not used in this sample?

Reply:
i meant to add the shortened version. thanks for noticing. :)